packages/medusa/src/services/fulfillment.ts

```diff
@@ -303,7 +334,7 @@ class FulfillmentService extends BaseService {
         trackingLinkRepo.create(tl)
       )
 
-      if (no_notification) {
+      if (typeof no_notification !== "undefined") {
```

Comment:
not related to this line, but this one:
https://github.com/medusajs/medusa/blob/d3061a75ea180ce41fa0d2b8845e5c49bb48ad04/packages/medusa/src/services/fulfillment.ts#L346

missing `await`

Reply:
indeed, it should be `return await ...`